tox.ini

```diff
@@ -28,6 +28,8 @@ deps =
 whitelist_externals = make
 
 commands =
+    mastereventlet: pip install --upgrade https://github.com/eventlet/eventlet/archive/master.zip
```

Comment:
Apparently it's much faster, presumably because it doesn't have to clone the whole repo. See http://stackoverflow.com/a/24811490/128749